### DISCUSSION TITLE
A diversidade precisa ser pensada em diversos recortes

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#112] [Talks in Portuguese]
**Palestrante**: Fernanda Souza
**Assunto**: Atualmente, muito se fala em diversidade, mas exatamente o que é essa diversidade? Você sabia que está além de pensar a inclusão de pessoas pretas e LGBTQIA+ no mercado de tecnologia? Precisamos pensar em todos os recortes de diversidade, em tudo que sai desse padrão social e incluir as mães solo, deficientes, periferias, pessoas mais velhas (+40) e por aí vai. Que tal vir ouvir e contribuir com essa discussão?

**Comment by leitoraincomum** [comment]
[Slides da palestra]( https://www.canva.com/design/DAFMTbLXGiQ/TpzrVSVA4FWNChQVIFgkIA/view?utm_content=DAFMTbLXGiQ&utm_campaign=designshare&utm_medium=link&utm_source=publishsharelink)
[LinkedIn ](https://www.linkedin.com/in/leitoraincomum/)
e-mail: fernanda@leitoraincomum.com.br
Demais redes: [beacons.page/leitoraincomum](beacons.page/leitoraincomum)

Muito obrigada por terem visto a palestra.
